packages/gatsby-image/src/__tests__/index.js

```diff
@@ -45,6 +87,32 @@ const setup = (fluid = false, onLoad = () => {}, onError = () => {}) => {
   return container
 }
 
+const setupImages = (
```

Comment:
It seemed easier to add this as a separate `setupImages` function, but I could also refactor the original `setup` to handle this case too.